strawberryfields/gbs/sample.py

```diff
@@ -216,14 +230,123 @@ def uniform_sampler(modes: int, sampled_modes: int, samples: int = 1) -> list:
     return output_samples
 
 
-def random_seed(seed: int = None) -> None:
+def seed(value: Optional[int]) -> None:
     """Seed for random number generators.
 
     Wrapper function for `numpy.random.seed <https://docs.scipy.org/doc/numpy//reference/generated
-    /numpy.random.seed.html>`_ to seed NumPy-based random number generators used in
-    :func:`quantum_sampler` and :func:`uniform_sampler`. This allows for repeatable sampling.
+    /numpy.random.seed.html>`_ to seed NumPy-based random number generators used in the GBS
+    applications layer. This allows for repeatable sampling.
```

Comment:
I think this might be a bit deceptive, it makes it sound like this seed will only affect functions in the GBS applications layer. However, settings this seed will affect _all_ numpy random functions.

Reply:
Great, have updated now to "to seed all NumPy-based random number generators. This allows for repeatable sampling"

Reply:
:+1: I think this is an argument for removing the wrapped 'seed' functions in `gbs` as well as `thewalrus`, as functions should never have surprising side-effects for users. But this can be considered later.